packages/asc-ui/src/components/Footer/FooterHeading.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components'
+import Heading from '../Heading/Heading'
+import { themeColor, themeSpacing } from '../../utils'
+
+const FooterHeading = styled(Heading)`
+  margin-top: 0;
+  margin-bottom: ${themeSpacing(3)};
+  color: ${themeColor('tint', 'level1')};
+  background-color: transparent;
+  display: flex;
+`
+
+FooterHeading.defaultProps = {
```

Comment:
I believe you could also accomplish this using the `.attrs({ forwardedAs: 'h3', styleAs: 'h2' })`, also should this not be `as` instead of `forwardedAs`?